tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -419,13 +419,10 @@ File and sync features in a shared scenario
 - [apiSharees/sharees.feature:430](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L430)
 - [apiSharees/sharees.feature:431](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L431)
 
-#### User cannot create a folder named Share
 
+#### [accepting matching name shared resources from different users/groups sets no seial identifiers on the resource name for the receiver](https://github.com/owncloud/ocis/issues/4289)
```

Comment:
What is "seial"? What did you intend to write?

Reply:
i mean `serial`